src/components/QuotesText.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import Image from "next/image";
+
+
+
+type Quotes = {
+  title: string;
+  quote: string;
+  imageDimensions: [number, number];
+}[];
+
+type QuoteTextProps = {
+  Quotes: Quotes;
+  buttonHighlighted: number;
+  setQuotesText: React.Dispatch<React.SetStateAction<string | string>>;
+  setButtonHighLighted: React.Dispatch<React.SetStateAction<number>>;
+};
+
+export default function QuotesText({
+  Quotes,
+  buttonHighlighted,
+  setQuotesText,
+  setButtonHighLighted,
+}: QuoteTextProps) {
+  const handleHighlightedButtons = (index: number) => {
+    setButtonHighLighted(index);
+  };
+
+  return (
+    <div className="flex-row flex items-center line-block ">
+      {Quotes.map((text, index) => {
+        return (
+          <>
+            <span
+              className={
+                buttonHighlighted === index
+                  ? "hover:text-white text-white text-center text-xl border-b-2 inline-block py-10 px-[10px] w-[280px] h-[132px]"
+                  : "text-gray-400 border-gray-400 text-center hover:border-white border-b-2 text-xl inline-block py-10 px-[10px] w-[280px] h-[132px]"
```

Comment:
There is duplication here that's not necessary. Whether it's the selected button right now or not, you're still setting certain classes regardless. py-10, px-[10px] etc

Reply:
this does not appear to be the case, when I remove styling from either side it will affect the styling.